jwst/assign_wcs/util.py

```diff
@@ -658,6 +677,17 @@ def create_grism_bbox(input_model,
                     xmax = int(np.max(xstack))
                     ymin = int(np.min(ystack))
                     ymax = int(np.max(ystack))
+                    if wfss_extract_half_height is not None:
```

Comment:
Is the source catalog info for each source known at this point in the processing? This param value needs to be applied ONLY if the source being processed has `is_star = True` in the source catalog (only applied to point sources).

Reply:
yes, the source catalog is available, will include this check